### PR TITLE
CinemaHall, ShoppingCart, User and Order controllers with dto's added

### PR DESCRIPTION
**Pull request by DmGrt**

### CODE REVIEW

File: src/main/java/com/spring/cinema/models/dto/shoppingcart/ShoppingCartDtoMapper.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.spring.cinema.models.dto.shoppingcart;
+
+import com.spring.cinema.models.ShoppingCart;
+import com.spring.cinema.models.Ticket;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShoppingCartDtoMapper {
+    public ShoppingCartResponseDto mapToDto(ShoppingCart order) {
```
Comment (RomanHreits) [inline src/main/java/com/spring/cinema/models/dto/shoppingcart/ShoppingCartDtoMapper.java]: the parameter should be renamed

**Review by RomanHreits — APPROVED** [review]
Good job!

File: src/main/java/com/spring/cinema/models/dto/shoppingcart/ShoppingCartDtoMapper.java (hunk 1 above)
Comment (KseniiaMakarova) [inline src/main/java/com/spring/cinema/models/dto/shoppingcart/ShoppingCartDtoMapper.java]:
```suggestion
    public ShoppingCartResponseDto mapToDto(ShoppingCart cart) {
```

File: src/main/java/com/spring/cinema/models/dto/user/UserResponseDto.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.spring.cinema.models.dto.user;
+
+import lombok.Data;
+
+@Data
+public class UserResponseDto {
+    private Long id;
+    private String email;
+    private String password;
```
Comment (KseniiaMakarova) [inline src/main/java/com/spring/cinema/models/dto/user/UserResponseDto.java]: let's not return secure data

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**